src/index.ts

```diff
@@ -349,6 +350,7 @@ export async function install(
       !!isOptimized && rollupPluginTreeshakeInputs(installTargets),
       !!isOptimized && rollupPluginTerser(),
       !!withStats && rollupPluginDependencyStats(info => (dependencyStats = info)),
+      ...userDefinedRollup.plugins, // load user-defined plugins last
     ],
     onwarn: ((warning, warn) => {
       if (warning.code === 'UNRESOLVED_IMPORT') {
```

Comment:
a feature that I'll suggest once we get this PR merged: 

Instead of a generic warning, it would be nice to suggest a plugin based on the file extension that we encounter. I think we already have a pretty good list:

- `.svelte`
- `.css` 

Similarly, our check for `isNodeBuiltin()` could now recommend a Node builtin plugin instead of just yelling at you about it.

Reply:
I can think of, like, 8 open issues/PRs/discussions that this would solve for :)

Reply:
Very cool idea!

Reply:
Made a quick log update for node builtins here: https://github.com/pikapkg/snowpack/commit/41a1c45897d3a37ee9b16a3ea7954c56e83a1f35

Will spin off the "by-file-extension" improved warnings to an issue.